### ISSUE TITLE
Conflicting instructions - install with admin/install without admin

### ISSUE THREAD

**Issue by illumineawake** [Asteski/WinMac#31]
Installation process could be clearer. The instructions are very clear that it must be done with admin, but then nexus dock cannot be installed when ran as admin. What is the process to follow here? Are you meant to go and re-run a custom install without admin (after the first admin install) and only select nexus?

Doing this has resulted in my nexus dock not following the theme or having separators, and yes I followed the install guide that says this is an issue and go to nexus preferences -> themes and press apply. This didn't make a difference :(

I also still have the windows taskbar showing after install, is manually hiding part of the process?

**Comment by akamienski** [comment]
@illumineawake 

Yes exactly, I'd recommend to install every component except Nexus as admin, and then just run another non-admin terminal session, and install Nexus from there.

I think I may consider blocking Nexus installation in admin session to prevent any issues, and for users to just allow to select full installation, without worrying about Nexus.

That separator thing only fixes that, and not other issues that may arise from installing Nexus as admin. Still - it's easily fixable. You can just uninstall it, and install it in non-admin session.

Apologies and hopefully that answers all your questions.

**Comment by illumineawake** [comment]
@akamienski thanks for the reply! My main call-out was that the current install instructions do not specify to run installer as admin (without nexus) and then run it again without admin as a custom installer to just install nexus. I think that would be helpful for new users that have never themed windows before (like me).

As for the taskbar issue I actually followed the process I described above i.e. installed everything as admin, except nexus because as you indicated it doesn't let you. Then installed nexus in a custom install without admin. At this point I ran into the problem of no separators and losing the theming of the dock. I could not seem to fix this, even going in and pressing apply on different themes etc. didn't resolve it. A full uninstall and reinstall fixed the issue, although I did not try just uninstalling nexus and reinstalling, I'm guessing that might have worked.

And I understand how you are using the Taskbar now after reading your comments and reviewing your screenshots. My immediate feeling, having been done away with the windows 7 style of listing open windows on the taskbar for some time now is I'd almost prefer to have it only display the windows icon (top left) and system tray (top right) and that's it. Do you know if there's a way to not display the open windows on that taskbar (it feels like a bit of duplication with the dock)? I've gone through all the settings and can't find this.

And last question (sorry for the essay), wondering with your experience using this, do you find any additional settings useful? e.g. do you auto hide doc or taskbar etc?

**Comment by akamienski** [comment]
@illumineawake It's not technically possiblew to hide running apps in Windows taskbar, you can use Rainmeter Droptop Four to achieve more linux-like menu bar. 

IMO it's not necesarily a duplication - I don't have any apps pinned to taskbar, so it only shows running apps - like tabs in browser, where dock is like a shelf from which I run apps, and I've also enabled auto-hide in dock, so I can only see it when I need it, and only see running apps in taskbar, which makes desktop not cluttered.

**Comment by illumineawake** [comment]
Thanks, this is looking really nice now. Couple more Qs:
1. You said above: "and only have running apps visible on desktop, which makes desktop feel very clean", I couldn't find this setting anywhere? Did you mean don't show runnings apps in dock?
2. On your point about the taskbar being like tabs, it makes a lot of sense, especially after auto-hiding dock. Is there any way to get middle clicking a window on the taskbar to make it close? Currently it's set to open a new window of that app which is the opposite behaviour to how middle clicking tabs works in my web browser. I looked at the middleclick button ahk but I am not familiar with AHK syntax. 
3. Lastly I can't work out how to unmap the windows key from the WinX menu? I am happy using ctrl+esc and it annoys me that if I use any other shortcut that involves the windows key it always also triggers the winx menu.

**Comment by akamienski** [comment]
@illumineawake 
1. I meant in taskbar, I don't have anything pinned there, so I can only see running apps.
3. Go to taskbar options for StartAllBack, and make sure that _combine buttons_ is set to never.
4. 3. You need to shutdown _windowskey.exe_ process and disable it in [Task Scheduler](https://github.com/Asteski/WinMac/wiki/Configuration#task-scheduler)

**Comment by illumineawake** [comment]
Thanks for the reply. Not sure if you understood my query in #2 or if that setting didn't work as you expected.
I was hoping I could middle click (scroll click) on an active program in the task bar and have it close the program. Currently middle clicking it starts a new window of that app. If not possible without significant edits no worries.

Lastly, I was wondering what is the update process like when you push a new update? Do I need to download and uninstall then re-install? Or can I just install over the top?

**Comment by akamienski** [comment]
1. Settings _combine taskbar buttons_ to never expire should allow you to close apps from taskbar with middle click.
2. No update capability is currently supported, you need to install specific components over the top, which would reset them to default settings.

**Comment by illumineawake** [comment]
Ok no worries, for what it's worth Setting combine taskbar buttons to never group does not allow me to close windows with middle click, as mentioned middle click seems to open a new window of the app, at least for me...

**Comment by akamienski** [comment]
@illumineawake that's weird, are you sure you selected **never group** instead of never?

**Comment by illumineawake** [comment]
@akamienski I don't see this option?
![image](https://github.com/user-attachments/assets/8cfef4db-7357-4abf-9c29-252ab93ee518)

**Comment by akamienski** [comment]
@illumineawake okay it seels you have an older version. Download 3.9 version [from here directly](https://startallback.com/), not from winget.

**Comment by illumineawake** [comment]
Perfect, I was able to update within startallback, found the option and now it works. Everything is pretty much perfect for me now, thanks so much!
